### DISCUSSION TITLE
How can I run a command that emits a open-api.json schema file?

### DISCUSSION THREAD

**Discussion by madhavgharmalkar** [vitalik/django-ninja#820] [Q&A] [answered]
👋🏾  - thanks for this awesome project!

I'd like to be able to run a command in the CLI that will generate and automatically output a open-api.json schema file. I would like to hopefully fun this file in my CI.

Thanks in advance!

**Comment by vitalik** [comment, marked as answer]
@madhavgharmalkar 

you just add ninja to installed apps:

```Python
INSTALLED_APPS = [
    ...
    'ninja',
]
```

and then you should be able to get it like this:

```
 python manage.py export_openapi_schema
```

**Reply by madhavgharmalkar** [reply]
Дякую!
